store/types/store.go

```diff
@@ -307,6 +314,12 @@ func (st StoreType) String() string {
 
 	case StoreTypeMemory:
 		return "StoreTypeMemory"
+
+	case StoreTypeSMT:
+		return "StoreTypeSMT"
+
+	case StoreTypeDecoupled:
+		return "StoreTypeDecoupled"
```

Comment:
what is this type?

Reply:
This represents the new `decoupled.Store` type